dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -2455,14 +2460,55 @@ Block MergingBuckets::getData(size_t concurrency_index)
 {
     assert(concurrency_index < concurrency);
 
-    if (unlikely(data.empty()))
+    if unlikely (data.empty())
```

Comment:
why removing the `()` outside of unlikely? 😂

Reply:
Just for simplicity😂